types/index.d.ts

```diff
@@ -2655,7 +2655,7 @@ export interface MathJsInstance extends MathJsFactory {
    */
   max(...args: MathScalarType[]): MathScalarType
   /**
-   * @param A A single matrix
+   * @param A An array of scalar values
```

Comment:
The argument can be either an array with scalar values (a 1-D matrix) or an array with arrays with scalar values (a 2-D matrix). I prefer to either keep these descriptions as they where, "A single matrix", or turn it into something like "A matrix" or "A 1-D or 2-D array with scalar values"